build-logic/jvm/src/main/kotlin/gradlebuild.unittest-and-compile.gradle.kts

```diff
@@ -336,7 +336,9 @@ fun removeTeamcityTempProperty() {
     }
 }
 
-fun Project.enableExperimentalTestFiltering() = !setOf("build-scan-performance", "configuration-cache", "kotlin-dsl", "performance", "smoke-test", "soak").contains(name) && isExperimentalTestFilteringEnabled
+fun Project.isPerformanceProject() = setOf("build-scan-performance", "performance").contains(name)
+
+fun Project.enableExperimentalTestFiltering() = !setOf("configuration-cache", "kotlin-dsl", "smoke-test", "soak").contains(name) && isExperimentalTestFilteringEnabled
```

Comment:
I think we shouldn't enable test selection for performance tests, yet. Let's see what the simulator tells us there.